ec2/spark_ec2.py

```diff
@@ -47,6 +48,8 @@
 else:
     from urllib.request import urlopen, Request
     from urllib.error import HTTPError
+    raw_input = input
+    xrange = range
```

Comment:
2.6/2.7 behavior is unaffected, the if-block just above is a check on py version number.  

In py3 xrange does not exist because range() is an iterator.  This is the approach that [six.py](https://code.google.com/p/googleappengine/source/browse/trunk/python/lib/django-1.5/django/utils/six.py?r=442#184) uses for porting.  

Similarly the eval behavior of py2's input was dropped for obvious reasons, and raw_input was basically renamed.  Six also directly ports these names a couple lines above the other link.